src/knd_class.json.c

```diff
@@ -179,9 +179,9 @@ static int export_conc_elem_JSON(void *obj,
 
     c->depth = 0;
```

Comment:
Why do we change class state in the export function?

Reply:
The depth and max_depth fields are the relics of a non-concurrent implementation of a kndClass. They are used as temporary variables to reflect the need for a further expansion of class attributes while traversal and serialization. The plan is to replace these fields with recurrent, thread-safe variables.